ci/gpu/build.sh

```diff
@@ -177,17 +171,16 @@ else
     gpuci_logger "Check GPU usage"
     nvidia-smi
 
-    set -x
-    cd $LIB_BUILD_DIR
+    gpuci_mamba_retry install -y -c ${CONDA_ARTIFACT_PATH} libcudf_tests
```

Comment:
This suggestion is unnecessary, but might improve transparency about the fact that `libcudf` and `libcudf_kafka` are also getting installed here:

```suggestion
    gpuci_mamba_retry install -y -c ${CONDA_ARTIFACT_PATH} libcudf libcudf_kafka libcudf_tests
```

Mostly suggesting because it took me a few seconds to wrap my head around why we were able to remove their installations below 😅